float64 r 
float64 theta
float64 timer
float64 angular
float64 linear_x
float64 linear_y
int32 obstacles
int32 timeout
